tests/framework/db/QueryBuilderTest.php

```diff
@@ -1024,6 +1024,7 @@ public function conditionProvider()
 
             // between
             [ ['between', 'id', 1, 10], '[[id]] BETWEEN :qp0 AND :qp1', [':qp0' => 1, ':qp1' => 10] ],
+            [ ['between', '[[id]]', 1, 10], '[[id]] BETWEEN :qp0 AND :qp1', [':qp0' => 1, ':qp1' => 10] ],
```

Comment:
Yes the construction like '{{tableName}}.[[id]]'` ignored in the tests. 
I had complexities when I make changes in commits like #12678